integration/testdata/secrets.json.golden

```diff
@@ -110,7 +112,49 @@
               }
             ]
           },
-          "Match": "echo ********"
+          "Match": "echo ********",
+          "Deleted": false,
+          "Layer": {}
+        }
+      ]
+    },
+    {
+      "Target": "sample.pem",
```

Comment:
Do you have any specific reason to add this key? This test case already has GitHub PAT and AWS access keys. Aren't they enough?